### DISCUSSION TITLE
Who maintains the Microsoft.VisualBasic nuget packag?

### DISCUSSION THREAD

**Discussion by jrmoreno1** [dotnet/runtime#80989] [Q&A]
The latest release of  https://www.nuget.org/packages/Microsoft.VisualBasic/10.4.0-preview.18571.3 is, from 4 years ago, a preview, and missing quite a bit. Any idea how I could contribute to fixing those issues?

**Comment by KalleOlaviNiemitalo** [comment]
Source code of Microsoft.VisualBasic/10.4.0-preview.18571.3 is at <https://github.com/dotnet/corefx/tree/57358b7f7214c233e8397483a9afc1698ac00c5b/src/Microsoft.VisualBasic> ([migrated to dotnet/runtime](https://github.com/dotnet/runtime/tree/1b9ea976896b2124d00c90f397f6d8a694ff121b/src/libraries/Microsoft.VisualBasic)), which matches the tag [v3.0.0-preview.18571.3](https://github.com/dotnet/corefx/releases/tag/v3.0.0-preview.18571.3). The Microsoft.VisualBasic project was later renamed to Microsoft.VisualBasic.Core in <https://github.com/dotnet/corefx/pull/34087> ([migrated to dotnet/runtime](https://github.com/dotnet/runtime/commit/16e30eb036885be519ad984c5d44f447525b539b)), but there is no Microsoft.VisualBasic.Core package in NuGet Gallery; I think the binaries are now shipped in .NET Runtime only.
